articles/clojure_elixir_mstalker/article.md

```diff
@@ -0,0 +1,149 @@
+# Elixir vs Clojure
+
+I've spent a bit of time thinking through what language to invest more time in: Elixir or Clojure. They're very similar in many ways. Both support safe concurrency. Both are functional and favor immutability. Clojure and Elixir are both somewhat niche languages. If you're deciding to learn or use one over the other, here are some considerations. Other people have weighed in on this choice [here](https://elixirforum.com/t/opinions-on-elixir-vs-clojure/208), [here](https://www.reddit.com/r/Clojure/comments/5wxgko/elixir_or_clojure/), and [here](https://smashingboxes.com/blog/choosing-your-future-tech-stack-clojure-vs-elixir-vs-go/), but I wanted to offer my own observations.
+
+
+## VM
+
+Elixir [compiles down to BEAM byte code](https://elixir-lang.org/crash-course.html#adding-elixir-to-existing-erlang-programs). Clojure [runs on the Java Virtual Machine](https://clojure.org/about/rationale) (JVM). If you need one virtual machine or the other, your choice between Elixir and Clojure has already been made.
+
+
+## Community
+
+My impression is that most people using either language have made an intentional choice. Most people in both seem excited about their languages.
+
+Both communities have Slack groups ([Elixir](https://elixir-slackin.herokuapp.com/), [Clojure](https://clojurians.herokuapp.com/)). Both have active forums ([Elixir](https://elixirforum.com/), [Clojure](https://groups.google.com/forum/#!forum/clojure)).
```

Comment:
might be worth mentioning the elixir IRC since that's where all the elixir core team hangs out, but the slack is a better resource for overall community I suspect, dealers choice of course, just thought I'd mention it since I've never seen jose on slack and he's always on IRC ( irc://irc.freenode.net/elixir-lang )